src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/_layout.scss

```diff
@@ -201,6 +201,12 @@ depends on a lot of factors:
 
 .l-header-right {
     float: right;
+
+    @media (max-width: $moving-column-single-width-min) {
+        i[class*=icon-] {
+            display: none;
+        }
+    }
```

Comment:
minShowWidth is used in css kind of. When A3 was designed - when you have 2 columns the header width remains a 1 column width as opposed to stretching out, this would look weird on a wide monitor